tests/common/util/TestObject.cpp

```diff
@@ -1166,6 +1166,30 @@ createMpTokenObject(std::string_view accountId, ripple::uint192 issuanceID, std:
     return mptoken;
 }
 
+ripple::STObject
+createPermissionedDomainObject(
+    std::string_view accountId,
+    ripple::LedgerIndex ledgerIndex,
```

Comment:
ripple::LedgerIndex is a uint32_t which represents ledger sequence. Here sfLedgerIndex expects a uint256, we usually pass it via string_view. Ledger sequence can not be used as index of ledger entry. I know the name is confusing 😵